xbmc/video/VideoDatabase.h

```diff
@@ -67,6 +67,18 @@ namespace VIDEO
   struct SScanSettings;
 }
 
+enum EDetailsFlag
```

Comment:
We don't prefix enums with an `E`. And IMO `DetailsFlag` is a bit generic. Maybe something like
```
enum class VideoDbDetails
{
  None = 0x00,
  Rating = 0x01,
  ...
  All = 0xFF
};
```
That way you'll have to use `VideoDbDetails::None` etc which is IMO more obvious. And having `All` as `0xFF` is more future proof.